airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -213,6 +213,14 @@
   icon: confluence.svg
   sourceType: api
   releaseStage: alpha
+- name: Convex
+  sourceDefinitionId: c332628c-f55c-4017-8222-378cfafda9b2
+  dockerRepository: airbyte/source-convex
+  dockerImageTag: dev
```

Comment:
how do we generate the docker image? the image tag should be 0.1.0 when we do so, i think

Reply:
The image is generated during the review process. Someone will trigger CI actions to test and publish the docker image to Dockerhub.
Yes. the image must be tagged 0.1.0